### DISCUSSION TITLE
Bluetooth 5.0 extended length advertisement

### DISCUSSION THREAD

**Discussion by gerdavax** [micropython/micropython#10948] [Libraries & Drivers]
Hi. I was not able to understand whether MicroPython supports extended length advertising on hardware that supports Bluetooth 5.0 or not. Is is supported? Thank you.

**Comment by jimmo** [comment]
@gerdavax Micropython does not support extended advertisements. This may be something we add in the future but it is not currently on the roadmap.

**Comment by Pawelinode** [comment]
Are planned support of Coded PHY frames in scanning function  (ESP32_C3, ESP32_S3) ?
